### DISCUSSION TITLE
provider use wagmi chains show error log

### DISCUSSION THREAD

**Discussion by yasoll** [rainbow-me/rainbowkit#1606] [Q&A]
I'm using Rainbowkit in NextJs13 and exported the wagmi chians in the config file , but it reports an error on startup
Error: It's currently unsupport to use "export *" in a client boundary. Please use named exports instead.

``` ts
import '@rainbow-me/rainbowkit/styles.css';
import {getDefaultWallets} from '@rainbow-me/rainbowkit';
import { configureChains, createConfig } from 'wagmi';
import {mainnet} from 'wagmi/chains';
import { publicProvider } from 'wagmi/providers/public';
export const { chains, publicClient } = configureChains(
  [mainnet],
  [publicProvider()]
);
const { connectors } = getDefaultWallets({
  appName: 'demo',
  projectId: 'b22f893d9d514d3c78f851e6552242de',
  chains
});
export const wagmiConfig = createConfig({
  autoConnect: true,
  connectors,
  publicClient
})
```

<img width="895" alt="image" src="https://github.com/rainbow-me/rainbowkit/assets/127667530/946c0cc7-8168-4dbc-900d-7b9f1147aa21">

**Comment by 0xmkh** [comment]
@yasoll What does your `tsconfig.json` file look like ? We've had similar issues with modules not supporting `export * from ...`
